### ISSUE TITLE
[Accessibility] Wizard navigation content reads additional information

### ISSUE THREAD

**Issue by devika-devanand** [SAP/fundamental-ngx#10327]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
On using the voice reader for accessing the page content within a wizard, additional information "wizard region” is being read. Information, properties of all UI elements should be conveyed correctly once to the screen reader users when accessed and additional information should be avoided.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

0.43.0-rc.16

#### If this is a bug, please provide steps for reproducing it.
-Use a voice reader and navigate through the fd-wizard-navigation UI
-Listen to the audio for the content UI frame.

#### Is there anything else we should know?

Screenshot:
![image](https://github.com/SAP/fundamental-ngx/assets/71935676/07fb24e1-cc85-4e8a-a751-a047525f2906)

**Comment by nikolay-kolarov** [comment]
Hi @devika-devanand,

This announcement comes, as the region you enter is called a Wizard region.
It is the same in UI5 following our UX recommendations https://ui5.sap.com/#/entity/sap.m.Wizard/sample/sap.m.sample.Wizard.
It is strange that with VoiceOveir it is announced on the focus inside the first step content, as with Jaws it is announced when you focus the first item in the navigation.
Still, such announcement is expected, so I propose to close this incident.

@droshev - FYI

Thanks,
Nikolay
